specifyweb/frontend/js_src/lib/components/FormEditor/Create.tsx

```diff
@@ -261,7 +266,13 @@ function ChooseName({
         onSubmit={(): void => {
           const uniqueName = getUnique(name);
           setViewSets(
-            createViewDefinition(viewSets, uniqueName, table, template),
+            createViewDefinition(
+              viewSets,
+              uniqueName,
+              table,
+              template,
+              newName
+            ),
```

Comment:
why pass the name to createViewDefinition and have it pass it right back when you can just use it here right away?

```suggestion
            {
              ...createViewDefinition(viewSets, uniqueName, table, template),
              name: viewSets.name === '' ? appResource.get('name') : viewSets.name,
            }